contributing/DEVELOPERS.md

```diff
@@ -250,17 +250,26 @@ Netlify is connected to the following github triggers:
 
 A new production version of kepler.gl website is automatically created and deployed every time a PR is merged onto master.
 
-In order to support testing environment, NEtlify is setup to generate build every time a PR is created or updated.
+In order to support testing environment, Netlify is setup to generate build every time a PR is created or updated.
 By generating builds for new and updated PRs we support CI/CD so developers can test their own build in a production like environment 
 
 ### Publish kepler.gl package to NPM
-In order to publsih a new verion of the kepler.gl package a developer must perform the following steps:
+
+#### Requirements
+To prepare a new release you need the following tool:
+- [gh-release](https://www.npmjs.com/package/gh-release): this tool facilitates the creation of a new git tag (using package.json version number) and a github release (different from npm release)
+
+Setup ```gh-release``` with your github api token ([instructions](https://www.npmjs.com/package/gh-release#command-line-interface))
+
+### Push a new release
+In order to publish a new version of kepler.gl a developer must perform the following steps:
 1. Update __package.json__ file with the new version value.
 2. Update each of the example folder package.json kepler.gl dependency with the newer version specified during step 1.
```

Comment:
example version should be updated after the release for build to succeed with script `npm run example-version`